glossary.yml

```
@@ -6060,8 +6060,8 @@
   ja:
     term: "変数 (プログラム)"
     def: >
-      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来ます。
-      変数は英語で「variable」(ヴァリアブル)と呼んでいます。
+      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来のこと。
```

Suggestion: プログラム内で使用される、データが関連付けられる箱の総称。変数の値は定義した後で変更が可能。

---

I get what they're going for with the 箱(box) metaphor to store the data. It's an interesting idea since it's a hard concept to explain and takes a long time for beginner's to understand. I've removed it for consistency with other languages. e.g., 

```diff
-    A name for box in a program that has some data stored in it. 
+    A name in a program that has some data associated with it. 
```

```diff
-    プログラム内で使用される、データが関連付けられる箱の総称。
+    プログラム内で使用される、データが関連付けられる総称。
```

Otherwise I agree to the suggestions. Your second sentence is nice and concise.

```suggestion
      プログラム内で使用される、データが関連付けられる総称。変数の値は定義した後で変更が可能。
```